backend/src/serverless/microservices/nodejs/automation/workers/newMemberWorker.ts

```diff
@@ -103,7 +103,7 @@ export default async (
       log.info(`Found ${automations.length} automations to process!`)
 
       let member: any | undefined = memberData
-      if (member === undefined) {
+      if (memberId) {
```

Comment:
can we do a better check?
if `memberData` is there why do we need to load it?

so something like this:
```javascript
if (memberData) // use member 
else if (memberId) // use memberId to load member
else // log and throw an error...
```